practice/src/main/java/practice/tasks/multithreading/MultithreadingTask.java

```diff
@@ -1,11 +1,42 @@
-package practice.tasks;
+package practice.tasks.multithreading;
 
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
 
 public class MultithreadingTask {
 
+    public int elementsSumWithThreadExecutor(int[] numbers, int threads) throws ExecutionException, InterruptedException {
+        int resultElementsSum = 0;
+        ExecutorService executor = Executors.newFixedThreadPool(threads);
```

Comment:
`ExecuterService` is an `AutoClosable` resource. Please handle it as needed and investigate why we need to close it.